src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageWithPropertiesExtensions.cs

```diff
@@ -65,14 +66,15 @@ public static class LoggerConfigurationAzureTableStorageWithPropertiesExtensions
 			bool writeInBatches = false,
 			TimeSpan? period = null,
 			int? batchPostingLimit = null,
-			string additionalRowKeyPostfix = null)
```

Comment:
This will be a breaking change, so a major version bump may be required depending on how we are doing versioning